src/cmd-build

```diff
@@ -219,7 +219,7 @@ img_base=tmp/${imageprefix}-base.qcow2
 # 9pfs isn't an option for EL7 so copying out anaconda logs doesn't work
 extraargs=
 if [ -n "${ISFEDORA}" ]; then
-    extraargs="${extraargs} --logs ${PWD}/tmp/anaconda"
+    extraargs="${extraargs} --fs9p"
```

Comment:
Do we even need this conditional anymore? This should work even on an el8 based cosa, right?

Reply:
> This should work even on an el8 based cosa, right?

RHEL8 kernel doesn't enable 9p either.
```
# grep 9P /usr/lib/modules/4.18.0-80.el8.x86_64/config 
# CONFIG_NET_9P is not set
```